### ISSUE TITLE
Fees calculation

### ISSUE THREAD

**Issue by gbarkhatov** [babylonchain/simple-staking#23]
At this point of time we are using predefined static `staking`, `unbonding` and `withdrawal` fee.

- When we create staking tx, we check the `utxos + staking fee`
- We can get exact size of the Transaction after we sign it and finalize (using the UI of the extension)

To get the idea about staking fee, we need to call the wallet `getNetworkFees` method, which returns us something like:

```ts
{
  fastestFee: 1,
  halfHourFee: 1,
  hourFee: 1,
  economyFee: 1,
  minimumFee: 1
}
```

Which is if I get it correctly is satoshis per vbyte.

Staking tx size is ~235, withdrawal tx is ~133. But we already put the fees when we were constructing them. What should we do?

I cannot do this on PSBT creation step like this because it will throw `Not finalized` error:
```ts
Psbt.fromHex(unsignedStakingTx.toHex())
  .extractTransaction()
  .virtualSize()
```

Should we define `X` average size of `staking`, `unbonding` and `withdrawal` transactions and then put a multiplication result that we get using `x satoshis per vbyte`?

_Originally posted by @gbarkhatov in https://github.com/babylonchain/simple-staking/pull/22#discussion_r1554644440_

**Comment by jrwbabylonchain** [comment]
Spend quite sometime looking into this lib codebase today. Don't feel like we can use its default `selectCoin` method.

What i'm concerned with is their methods will do some input selection based on the fees. this one https://github.com/bitcoinjs/coinselect/blob/master/index.js#L15 this might add unnecessary complexity into our codebase as this part of the code is on very critical path. 

However, the accumulative.js method they have should offers what we need by adding/calculating all inputs https://github.com/bitcoinjs/coinselect/blob/master/accumulative.js

**Comment by jrwbabylonchain** [comment]
we wil go with the `numInputs*180 + numOutputs*34 + 10 + numInputs + 40` approach to start with and evaluate based on the result from testnet.

This issue will keep open

**Comment by gbarkhatov** [comment]
Isn't it magic? i.e. 180, 34, 10, 40

**Comment by gbarkhatov** [comment]
Okay then

**Comment by gbarkhatov** [comment]
Closed in #185
